.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.DS_Store
```

Comment:
What is this?

Reply:
This is a macOS system file. It resides in every directory. I did not want this to be pushed alongside the code so I added it to gitignore.